lib/handler/console.js

```diff
@@ -40,7 +40,7 @@
       }
 
       console.error(logValues);
-    } else {
+    } else if (typeof logAttributes.msg !== 'undefined') {
```

Comment:
This doesn't quite do what I would want. If the `msg` attribute is missing I'd still prefer we log out what has been added to the logAttributes via all the `Appenders` that have been ran.  

I think we'd get more utility if we changed up how this handler worked in regards to the nicely formatted error messages. 

This will likely mean creating at least one reserved attribute for the object that is output from all the `Appenders`.  That attribute should have a `message` added to it from the `exception` that is raised _or_ from the `message` that is passed to the `Canadarm.LOG_LEVEL` call.  

If we do not want to go that deep on this issue, I would prefer this code snippet change to: 

```js
else if (typeof logAttributes.msg !== 'undefined') {
  console.error(logAttribugtes.msg, logAttributes);
} else {
  console.error(logAttributes);
}
```

In either scenario `console` should always get called and produce something.